<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>553</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="217">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="217">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Кошачий фотоальбом</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="btn_right">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/chevrons/images/chevron_r.png</normaloff>:/chevrons/images/chevron_r.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="btn_left">
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/chevrons/images/chevron_l.png</normaloff>:/chevrons/images/chevron_l.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Период переключения</string>
     </property>
     <addaction name="action_0sec"/>
     <addaction name="action_1sec"/>
     <addaction name="action_5sec"/>
     <addaction name="action_10sec"/>
    </widget>
    <addaction name="action_up_windows"/>
    <addaction name="action_use_resources"/>
    <addaction name="action_choose_dir"/>
    <addaction name="menu_2"/>
    <addaction name="close_programm"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_up_windows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Поверх всех окон</string>
   </property>
  </action>
  <action name="action_use_resources">
   <property name="text">
    <string>Фото из файла ресурсов</string>
   </property>
  </action>
  <action name="action_choose_dir">
   <property name="text">
    <string>Фото из выбранной папки</string>
   </property>
  </action>
  <action name="action_0sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Отключить</string>
   </property>
  </action>
  <action name="action_1sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 секунда</string>
   </property>
  </action>
  <action name="action_5sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5 секунд</string>
   </property>
  </action>
  <action name="action_10sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10 секунд</string>
   </property>
  </action>
  <action name="close_programm">
   <property name="text">
    <string>Закрыть программу</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close_programm</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
